spec/controllers/catalog_controller_spec.rb

```diff
@@ -83,8 +83,9 @@
        "repository_tesim", "rights_country_tesim", "rights_holder_tesim", "rubricator_tesim",
        "scribe_tesim", "script_tesim", "services_contact_ssm", "subject_tesim", "subject_topic_tesim",
        "summary_tesim", "support_tesim", "title_tesim", "toc_tesim", "uniform_title_tesim",
-       "writing_and_hands_tesim", "writing_system_tesim", "commentator_tesim", "translator_tesim",
-       "license_tesim", "normalized_date_sim", "subject_geographic_tesim", "subject_temporal_tesim"]
+       "hand_note_sim", "writing_system_tesim", "commentator_tesim", "translator_tesim",
```

Comment:
run the test and change accordingly, maybe you need to remove delivery and image count fields and change shelfmark, form and other fields.